### ISSUE TITLE
Leading spaces get eliminated

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fracte/keepnote#399]
```
##### What steps will reproduce the problem?
Create a note with text (code in a programming language) such that spaces are 
used to align lines after a line break. In the following example, "xlab" and 
"main" is supposed to start below the "d" of "data1"

plot3d(data1, data2, data3,
       xlab="DAT1", ylab="DAT2", zlab="DAT3",
       main="ALL THE DATS")

##### What is the expected output? What do you see instead?
The expected output is shown above. What happens instead is that with every 
saving of the notebook, one more space gets eliminated before "xlab" and before 
"main".

##### What version of the product are you using? On what operating system?
0.6.7 on Ubuntu 10.10

```

Original issue reported on code.google.com by `stgr...@gmail.com` on 22 Jan 2011 at 10:10
- Merged into: #502

**Comment by GoogleCodeExporter** [comment]
Original comment by `matt.rasmus@gmail.com` on 23 Feb 2011 at 4:05
- Added labels: **editor**

**Comment by GoogleCodeExporter** [comment]
```
clarification: this particularly happens when one has leading spaces in one 
note (cf. <Screenshot1.png>), CHANGES TO A DIFFERENT NOTE (I just clicked on 
the next note below), and then just saves the notebook. When one then goes back 
to the first note, spaces have been eliminated (cf. <Screenshot2.png>). Still 
an issue in 0.7.4 on Mint 11.

```

Original comment by `stgr...@gmail.com` on 16 Jul 2011 at 7:44

Attachments:
- [Screenshot1.png](https://storage.googleapis.com/google-code-attachments/keepnote/issue-399/comment-2/Screenshot1.png)
- [Screenshot2.png](https://storage.googleapis.com/google-code-attachments/keepnote/issue-399/comment-2/Screenshot2.png)

**Comment by GoogleCodeExporter** [comment]
```
Sorry, I responded to the newer version (issue 502) of this issue instead of 
this one.  Bug will be fixed in keepnote 0.7.5.
```

Original comment by `matt.rasmus@gmail.com` on 16 Jul 2011 at 8:21
- Changed state: **Duplicate**
